test/DotnetDocument.Tests/DocumentationBuilderTests.cs

```diff
@@ -88,7 +123,47 @@ public void Test2()
                 .Build();
 
             // Assert
-            documentedNode.ToFullString().ShouldBe(ExpectedDocumentedMethodText);
+            documentedNode.ToFullString().ShouldBe(ExpectedDocumentedMethodTextBlockScope);
+        }
+
+        /// <summary>
+        /// Tests file scoped namespace
+        /// </summary>
+        [Fact(DisplayName = "Yes")]
```

Comment:
Tests look good.
Can we either drop or put a relevant names in the display name `[Fact(DisplayName = "Yes")]` for this test and the one above?
It might have been a typo from me but naming two tests `Yes` looks weird in the IDE :see_no_evil:

Reply:
Dang, that's what happens when you copy and paste late at night. I dropped the display name so the tests will now show the method name as the title, which are appropriately named.